### ISSUE TITLE
Jira Commit Hint throws exception for some Jira issues

### ISSUE THREAD

**Issue by haldyr** [gitextensions/gitextensions#8122]
## Current behaviour
We use cloud Jira and for some issues Jira Commit Hints fails to return list of issues in commit window. Instead it returns stacktrace:

```
System.NullReferenceException: Object reference not set to an instance of an object.
   at Atlassian.Jira.Remote.SingleObjectCustomFieldValueSerializer.FromJson(JToken json)
   at Atlassian.Jira.Remote.RemoteIssueJsonConverter.<GetCustomFieldValuesFromObject>b__9_1(JProperty field)
   at System.Linq.Enumerable.WhereSelectEnumerableIterator`2.MoveNext()
   at System.Linq.Buffer`1..ctor(IEnumerable`1 source)
   at System.Linq.Enumerable.ToArray[TSource](IEnumerable`1 source)
   at Atlassian.Jira.Remote.RemoteIssueJsonConverter.ReadJson(JsonReader reader, Type objectType, Object existingValue, JsonSerializer serializer)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.DeserializeConvertable(JsonConverter converter, JsonReader reader, Type objectType, Object existingValue)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.Deserialize(JsonReader reader, Type objectType, Boolean checkAdditionalContent)
   at Newtonsoft.Json.JsonSerializer.DeserializeInternal(JsonReader reader, Type objectType)
   at Newtonsoft.Json.JsonConvert.DeserializeObject(String value, Type type, JsonSerializerSettings settings)
   at Newtonsoft.Json.JsonConvert.DeserializeObject[T](String value, JsonSerializerSettings settings)
   at Atlassian.Jira.Remote.IssueService.<>c__DisplayClass19_0.<GetIssuesFromJqlAsync>b__0(JObject issueJson)
   at System.Linq.Enumerable.WhereSelectEnumerableIterator`2.MoveNext()
   at System.Linq.Enumerable.WhereSelectEnumerableIterator`2.MoveNext()
   at System.Linq.Buffer`1..ctor(IEnumerable`1 source)
   at System.Linq.Enumerable.ToArray[TSource](IEnumerable`1 source)
   at JiraCommitHintPlugin.JiraCommitHintPlugin.<GetMessageToCommitAsync>d__32.MoveNext()
```

## Expected behaviour
Jira Commit Hint should return list of Jira issues.

## Steps to reproduce
In Jira Commit Hint plugin set
- JQL Query: `assignee in (currentUser()) AND gh.sprint.customfield.default.name in openSprints() ORDER BY updatedDate`

Than some issue cannot be deserialized see screenshot.

## Screenshots
![image](https://user-images.githubusercontent.com/5140211/82371281-365b4600-9a1a-11ea-9074-cf1bc42a8486.png)

## Did this work in previous version of GitExtensions
I think bug is related to Atlassian.SDK

## Environment
- Git Extensions 3.3.1.7897
- Build 5a97671645532bcedc443bac7b727f40db47cb5c
- Git 2.23.0.windows.1
- Microsoft Windows NT 10.0.18363.0
- .NET Framework 4.8.4180.0
- DPI 96dpi (no scaling)

**Comment by haldyr** [comment]
i'll try to look at it.

**Comment by gerhardol** [comment]
Closed by #8123
